src/xSTIR/cSTIR/cstir.cpp

```diff
@@ -176,10 +176,8 @@ void* cSTIR_newObject(const char* name)
 			return NEW_OBJECT_HANDLE(RayTracingMatrix);
 		if (sirf::iequals(name, "SPECTUBMatrix"))
 			return NEW_OBJECT_HANDLE(SPECTUBMatrix);
-#if STIR_VERSION >= 050100
```

Comment:
probably fine to remove this since `STIR_REQUIRED_VERSION` in cmake is currently `6.0`